state/data/reducer.js

```diff
@@ -20,10 +21,13 @@ const data = (state = initialState, action) => {
         ...state,
         lists: action.lists
       };
-    case types.MERGE_LISTS_LIST:
+    case types.MERGE_MOVIES:
       return {
         ...state,
-        lists: action.lists
+        movie: {
```

Comment:
You don't need to have separate state for movie, you already have movies - this is the entry point for all movies which used in the app

Reply:
Is this okay to override(if it exists) movie short movie info fetched for list preview?